src/packer.pkr.hcl

```
@@ -46,88 +53,101 @@ variable "skip_create_ami" {
   type        = bool
 }
 
-data "amazon-ami" "debian_bullseye" {
+data "amazon-ami" "windows_server_2022" {
   filters = {
-    name                = "debian-11-amd64-*"
+    name                = "Windows_Server-2022-English-Full-Base-*"
     root-device-type    = "ebs"
     virtualization-type = "hvm"
   }
   most_recent = true
-  owners      = ["136693071363"]
+  owners      = ["amazon"]
   region      = var.build_region
 }
 
 locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
 
-source "amazon-ebs" "example" {
+source "amazon-ebs" "windows" {
   ami_block_device_mappings {
     delete_on_termination = true
     device_name           = "/dev/xvda"
     encrypted             = true
     volume_size           = 8
     volume_type           = "gp3"
   }
-  ami_name                    = "example-hvm-${local.timestamp}-x86_64-ebs"
+
+  ami_name                    = "windows-server-2022-${local.timestamp}-x86_64-ebs"
   ami_regions                 = var.ami_regions
   associate_public_ip_address = true
-  encrypt_boot                = true
-  instance_type               = "t3.small"
-  kms_key_id                  = var.build_region_kms
+
+  communicator  = "winrm"
+  encrypt_boot  = true
+  instance_type = "t3.large"
+
+  kms_key_id = var.build_region_kms
+
```

Remove unneeded blank lines:
```suggestion
  communicator  = "winrm"
  encrypt_boot  = true
  instance_type = "t3.large"
  kms_key_id = var.build_region_kms
```

---

Resolved in https://github.com/cisagov/windows-commando-packer/pull/10/commits/7dcca2527dabeaae77ffafe78cc506d0b2df18d0.